widgetastic_manageiq/__init__.py

```diff
@@ -5374,5 +5374,61 @@ def select(self, name):
         return False
 
 
+class RolesSelector(Widget):
+    """This widget for custom button role selection"""
+
+    ROOT = "//div[@class='col-md-10']//table"
```

Comment:
I don't think this is a good locator for this widget, as I'm not sure these classes will be very consistent. Is there nothing else in the DOM that can be used to identify this table?

Reply:
I know... you can say it was for initial implementation. I forgot to replace it with locator as an argument. Now locator is an argument with `table` tag.

yup, we don't have supporting things in `dom` but I would like to construct `XPath` with child elements support. I construct it with the help of `label` l.e. `User Roles`.
```
locator=".//label[contains(text(),'User Roles')]/../div/table"
```

Reply:
Since this is a widget specific to that selector, I also don't think its a good idea to force the view to pass that entire locator. The locator needed to be more accurate than just the div class, but since the widget is specific for MIQ it can be a ParametrizedLocator, where the constructor just takes the label text. Contains vs exact match could be an additional kwarg

Reply:
its also good practice to wrap any `text()` use in normalized-space() call as well.